bigquery/docs/index.rst

```diff
@@ -1,408 +1,22 @@
 .. include:: /../bigquery/README.rst
 
 Using the Library
```

Comment:
The flow is a bit awkward here - There is a heading for performing a query and then a separate heading for "Using the Library". I think the "Using the Library" heading should be updated to say something like "More Examples".
![image](https://user-images.githubusercontent.com/19769177/47112886-6c1bc580-d20c-11e8-8980-e3e88dbd27fb.png)

Reply:
I changed the heading as suggested and broke it up into logical sub-sections.